ckanext/oauth2/db.py

```diff
@@ -18,10 +18,17 @@
 # along with OAuth2 CKAN Extension.  If not, see <http://www.gnu.org/licenses/>.
 
 import sqlalchemy as sa
+import ckan.model.meta as meta
+import logging
```

Comment:
There is no logging in this file, except a `getLogger`. I suggest we discard this.